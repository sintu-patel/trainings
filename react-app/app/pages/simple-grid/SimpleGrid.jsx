import React from 'react';

class SimpleGrid extends React.Component {
	render() {
		return (
			<div className='simple-grid'>
				<div className='header'>
					Header
				</div>
				<div className='content'>
					Content
				</div>
				<div className='footer'>
					Footer
				</div>
			</div>
		);
	}
}
export default SimpleGrid;