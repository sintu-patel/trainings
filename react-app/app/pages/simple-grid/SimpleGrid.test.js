import React from 'react';
import {mount} from 'enzyme';
import renderer from 'react-test-renderer';
import SimpleGrid from './SimpleGrid.jsx';

describe('SimpleGrid', () => {

	test('Snapshot is equal to previous snapshot', () => {
        const tree = renderer
            .create(<SimpleGrid />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
    
    const wrapper = mount(<SimpleGrid />);
    test('SimpleGrid Renders', () => {
        expect(wrapper.find('.simple-grid').length).toBe(1);
    });
});
