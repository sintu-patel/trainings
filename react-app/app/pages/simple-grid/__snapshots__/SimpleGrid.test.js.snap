// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleGrid Snapshot is equal to previous snapshot 1`] = `
<div
  className="simple-grid"
>
  <div
    className="header"
  >
    Header
  </div>
  <div
    className="content"
  >
    Content
  </div>
  <div
    className="footer"
  >
    Footer
  </div>
</div>
`;
